fire/core.py

```diff
@@ -241,12 +241,17 @@ def _IsHelpShortcut(component_trace, remaining_args):
   return show_help
 
 
-def _PrintResult(component_trace, verbose=False):
+def _PrintResult(component_trace, verbose=False, serialize=None):
   """Prints the result of the Fire call to stdout in a human readable way."""
   # TODO(dbieber): Design human readable deserializable serialization method
   # and move serialization to its own module.
   result = component_trace.GetResult()
 
+  # Allow users to modify the return value of the component and provide 
+  # custom formatting.
+  if callable(serialize):
+    result = serialize(result)
```

Comment:
We'll want to handle the not-callable case too.
I think if we cannot call serialize, we raise a FireError.

Reply:
Maybe we just do:

```python
if serialize:
    result = serialize(result)
```
which will just raise whatever error the object would raise if tried to be called (probably typeerror)

Reply:
addressed - it will not raise `FireError("serialize argument {} must be empty or callable.".format(serialize))`